Xamarin.Forms.Core/NavigationPage.cs

```diff
@@ -4,8 +4,13 @@
 using System.ComponentModel;
 using System.Linq;
 using System.Threading.Tasks;
+using Xamarin.Forms;
 using Xamarin.Forms.Internals;
 using Xamarin.Forms.Platform;
+using Xamarin.Forms.StyleSheets;
+
+[assembly: StyleProperty("-xf-navigationpage-bar-background-color", typeof(NavigationPage), nameof(NavigationPage.BarBackgroundColorProperty))]
+[assembly: StyleProperty("-xf-navigationpage-bar-text-color", typeof(NavigationPage), nameof(NavigationPage.BarTextColorProperty))]
```

Comment:
drop the navigationpage

Reply:
Sure thing. But I'll wait for @jassmith to sound off.

Reply:
`-xf-bar-background-color`
`-xf-bar-text-color`